Lib/logging/__init__.py

```diff
@@ -1890,9 +1892,21 @@ def __init__(self, logger, extra=None):
         following example:
 
         adapter = LoggerAdapter(someLogger, dict(p1=v1, p2="v2"))
+
+        By default, LoggerAdapter objects will drop the "extra" argument
+        passed on the individual log calls to use its own instead.
+
+        Initializing it with merge_extras=True will instead merge both
+        maps when logging, the indivicual call extra taking precedence
```

Comment:
Typo `indivicual` -> `individual` and maybe quote 'extra' in the comment as in the previous paragraph.